WIP-SQL/connecting-to-a-remote-sql-server.md

```diff
@@ -0,0 +1,60 @@
+# Connecting to a remote SQL Server
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More on connecting to a remote server](https://enki.com)'
+
+---
+## Content
+
+Assuming the server is up and running, we will use `sqlcmd` utility to connect remotely:
```

Comment:
Have you tested this? Does this work in bash? I think this a command prompt only utility.